src/components/actionBar/filterSort/FilterSort.tsx

```diff
@@ -1,3 +1,261 @@
+                            ) : (
+                                <></>
+                            )}
+                        </Stack>
+                        <Button
+                            variant="outlined"
+                            id="bus-type-button"
+                            onClick={busTypeHandler}
+                            aria-describedby="bus-type-menu"
+                            aria-controls={
+                                openBusTypeFilter ? 'bus-type-menu' : undefined
+                            }
+                            aria-haspopup="true"
+                            aria-expanded={
+                                openBusTypeFilter ? 'true' : undefined
+                            }
+                        >
+                            Bus type
+                        </Button>
+                        <Button
+                            variant="outlined"
+                            id="seat-type-button"
+                            onClick={seatTypeHandler}
+                            aria-controls={
+                                openSeatTypeFilter
+                                    ? 'seat-type-menu'
+                                    : undefined
+                            }
+                            aria-haspopup="true"
+                            aria-expanded={
+                                openSeatTypeFilter ? 'true' : undefined
+                            }
+                        >
+                            Seat type
+                        </Button>
+                        <Button
+                            onClick={() => {
+                                setBusTypeFilter(null);
+                                setSeatTypeFilter(null);
+                            }}
+                        >
+                            Clear all
+                        </Button>
+                    </Stack>
+                    <Badge
+                        invisible={Boolean(!sortBy)}
+                        color="primary"
+                        badgeContent=" "
+                        variant="dot"
+                    >
+                        <Button variant="outlined" onClick={sortTypeHandler}>
+                            Sort
+                        </Button>
+                    </Badge>
+                </Stack>
+
+                {/* Filters menu */}
+                <Menu
+                    id="bus-type-menu"
+                    anchorEl={anchorElBus}
+                    open={openBusTypeFilter}
+                    MenuListProps={{ 'aria-labelledby': 'bus-type-button' }}
+                    onClose={handleMenuClose}
+                >
+                    {/* <FormControl> */}
+                    <RadioGroup
+                        onChange={busFilterHandler}
+                        value={busTypeFilter}
+                    >
+                        <StyledFormControlLabel
+                            label="AC"
```

Comment:
Maybe you can move these labels to constants file.

Reply:
All labels have been replaced using usetraslation.